### ISSUE TITLE
[Suggestion] Misc - Center Timer Display.

### ISSUE THREAD

**Issue by stunslave** [surftimer/Surftimer-olokos#192]
**Describe the solution you'd like**
As shown in the image below. If possible I would like a CVAR for **[ON] Center Timer Display** in Misc menu. 

Which will **add/show** the **Timer / Zone (Linear or Staged "Number" / Speed and the Pre-strafe speed** below the [ON] Center Speed Display. Something basic and easier to read like the image below.

**Additional context**
![image](https://user-images.githubusercontent.com/47533535/85859571-aac59980-b7bd-11ea-93f1-61eec33eeac1.png)

**Comment by qawery-just-sad** [comment]
if you type !timer -> center hud options you can customise almost everything

**Comment by stunslave** [comment]
> if you type !timer -> center hud options you can customise almost everything

Yes I know... but I'm not looking to use the current center hud!
I'm asking for a basic one to be added. Something that is easy to follow with a bigger timer display and not clodded with unneeded stuff. This way it is also centered with a clean look.

**Comment by qawery-just-sad** [comment]
So like !timer -> center hud options -> clean or custom and then clean would be something like that picture/surfheaven/ksf
and custom the way it is rn?
I propose to postpone that feature until v1.5, sounds good?

**Comment by stunslave** [comment]
> I propose to postpone that feature until v1.5, sounds good?
Ye this can be done whenever!

**Comment by Kyli3Boi** [comment]
I think this is a great idea as the current center hud is quiet large. Iv had to turn my hud scaling down to make it smaller which works but the downside is that the chat font size is very small
